src/components/common/constants.ts

```diff
@@ -79,11 +80,17 @@ export const ERROR_CODE_MAPPING: { [p: string]: string } = {
     SECURITY_CODE_ERROR,
     SecurityCodeErrorType.OtpMaxCodesSent
   ),
-  [ERROR_CODES.VERIFY_PHONE_NUMBER_CODE_REQUEST_BLOCKED]: pathWithQueryParam(
-    PATH_NAMES["SECURITY_CODE_WAIT"],
-    SECURITY_CODE_ERROR,
-    SecurityCodeErrorType.OtpBlocked
-  ),
+  [ERROR_CODES.VERIFY_PHONE_NUMBER_CODE_REQUEST_BLOCKED]: support2hrLockout()
```

Comment:
To me I'm still a bit unsure why this code change is needed when I read the commit message and given that I don't think there are any tests for this. To me this is about going to a different page when we have two hour lockout - why would we not now take them to the wait page?